java-manta-client-unshaded/src/main/java/com/joyent/manta/config/DefaultsConfigContext.java

```diff
@@ -91,6 +91,12 @@
      */
     public static final boolean DEFAULT_DISABLE_NATIVE_SIGNATURES = false;
 
+    /**
+     * The default depth of directories to prune to when prune depth
```

Comment:
s/depth of directories/number of ancestor directories/? (If we have a request to delete a directory, then the base directory is always deleted. I think it would just be good to emphasize that the verb 'prune' here refers the side effects of deleting an object/directory whose ancestors become empty after the fact).